DESCRIPTION

```diff
@@ -29,7 +29,11 @@ Imports:
     scales (>= 1.1.1),
     stringr (>= 1.4.0),
     tibble (>= 3.0.6),
-    tidyr (>= 1.1.2)
+    tidyr (>= 1.1.2),
+    future (>= 1.24.0),
+    furrr (>= 0.2.0),
```

Comment:
Ensure all packages added to DESCRIPTION are necessary for the final functionality to be merged